internal/cache/controller_obj_cache.go

```diff
@@ -988,6 +1007,9 @@ func (c *AviObjCache) AviPopulateOnePKICache(client *clients.AviClient,
 			continue
 		}
 		checksum := lib.SSLKeyCertChecksum(*pkikey.Name, *pkikey.CaCerts[0].Certificate, "")
+		if lib.GetEnableGRBAC() && pkikey.Labels != nil {
```

Comment:
@chauhanshubham: Each of this Checksum function have different calling points. For example, if we take into account "SSLKeyCertChecksum" ... it has been called while populating cache as well as at layer2 for computing checksum of object from k8 side. Similar scenarios for other component checksum calculations.  This will lead to complicated logic of checking calling point and then according to that add checksum value.

Reply:
ok sure.